build.gradle

```diff
@@ -12,7 +12,7 @@ plugins {
 }
 
 group 'uk.gov.hmcts.reform'
-version '2.3.1'
+version '2.3.2'
```

Comment:
maybe back to `2.3.2` and we can delete `2.3.1`?

Reply:
I don't have access to do, please let then know who ever have access to do

Reply:
WIP. we'll get back to it as soon as we can